tests/appsec/test_traces.py

```diff
@@ -119,6 +126,28 @@ def validate_response_headers(span):
         interfaces.library.validate_spans(self.r, validate_request_headers)
         interfaces.library.validate_spans(self.r, validate_response_headers)
 
+    @missing_feature(library="java", reason="Not implemented yet")
+    @missing_feature(library="dotnet", reason="Not implemented yet")
+    @missing_feature(library="python", reason="Not implemented yet")
+    @missing_feature(library="ruby", reason="Not implemented yet")
+    @missing_feature(library="php", reason="Not implemented yet")
+    @missing_feature(library="cpp", reason="Not implemented yet")
+    def test_extra_header_collection(self):
```

Comment:
you're missing a `def setupèextraèheader_collection` no ? the system-test is failing

Reply:
Yeah, ideally it should be a new setup method with a new request rather than reusing the previous one. Also, I would suggest naming this test something like `test_x_amzn_trace_id_header_collection`, since we'll add another header at some point, and we'll need to seggregate support metadata for it since it will be added later than this one.